### PR TITLE
Remove previews downloading when task creating with cloud storage data

### PR DESCRIPTION
**Pull request by Marishka17**
### Motivation and context

PR removes previews downloading from CS when task creating (skipped in PR #5478). In addition, I had to change the test to check for the file name existing in the message when the specified file is not found in the bucket, because now the preview is no longer downloaded at the stage of creating a task.
### How has this been tested?

Manually
### Checklist

- [x] I submit my changes into the `develop` branch
~~- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file~~
~~- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly~~
~~- [ ] I have added tests to cover my changes~~
~~- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
~~- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: tests/python/shared/utils/s3.py [hunk 1]
```diff
@@ -40,6 +42,11 @@ def file_exists(self, bucket: str, filename: str) -> bool:
             else:
                 raise
 
+    def download_fileobj(self, bucket: str, key: str) -> bytes:
+        data = BytesIO()
```
Comment (azhavoro) [inline tests/python/shared/utils/s3.py:46]: I would suggest using context manager here

File: tests/python/rest_api/test_tasks.py [hunk 2]
```diff
@@ -817,5 +824,28 @@ def test_cannot_create_task_with_mythical_cloud_storage_data(
...
+                    task_id, number=0, quality="original", type="frame"
+                )
+                raise AssertionError("Frame should not exist")
+            except AssertionError:
+                raise
+            except Exception as ex:
+                assert ex.status == HTTPStatus.NOT_FOUND
+                assert image_name in ex.body
+
+        s3_client.create_file(bucket_name, image_name, image)
```
Comment (azhavoro) [inline tests/python/rest_api/test_tasks.py:851]: Will the file be restored if the test fails?  Can this file affect other tests in this case?

**Comment by nmanovic** [comment]
@Marishka17 , could you please update the PR?

File: tests/python/rest_api/test_tasks.py (hunk 2 above)
Comment (Marishka17) [reply tests/python/rest_api/test_tasks.py]: Fixed

**Comment by nmanovic** [comment]
@azhavoro , could you please review the PR?

**Comment by Marishka17** [comment]
@nmanovic, @azhavoro, were you waiting for me to return from vacation to review my PR?:)

**Review by azhavoro — APPROVED** [review]

**Pull request merged**